lib/avromatic/schema_registry_patch.rb

```diff
@@ -0,0 +1,27 @@
+require 'avro_turf'
+require 'avro_turf/schema_registry'
+
+AvroTurf::SchemaRegistry.class_eval do
+  # Override register to first check if a schema is registered by fingerprint
+  def register(subject, schema)
+    schema_object = if schema.is_a?(String)
+                      Avro::Schema.parse(schema)
+                    else
+                      schema
+                    end
+
+    registered = false
+    data = begin
+             get("/subjects/#{subject}/fingerprints/#{schema_object.sha256_fingerprint.to_s(16)}")
```

Comment:
Should we provide a configuration option to opt out of this for anyone using the Confluent schema registry? Looks like the code would still work but will make an extra network request that will always fail when resolving a schema.